src/client/index.ts

```diff
@@ -466,9 +466,7 @@ export class Client {
 		}
 
 		if (this.config.autoEnableTokens && Object.keys(this.tokenStore.getSelectedTokens()).length)
-			this.eventSender('tokens-selected', {
-				selectedTokens: this.tokenStore.getSelectedTokens(),
-			})
+			this.eventSender('tokens-selected', this.tokenStore.getSelectedTokens())
```

Comment:
Let's keep the tokens data nested so we can allow backward compatibility without breaking changes:
```
{
   tokens: { [collectionId: string]: (DecodedToken|any)[] } // Any union for now until we formalise on-chain token interface
   // then we can add these later
   tokenCount: number,
   collectionCount: number
}
```

Also please update the event sender interface to match the new structure.

Reply:
Updated the PR to keep the namespace selectedTokens.